solutions/src/main/java/ru/sberbank/school/task02/Converter.java

```diff
@@ -0,0 +1,54 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.exception.FxConversionException;
+import ru.sberbank.school.task02.util.*;
+
+import java.math.BigDecimal;
+import java.util.Comparator;
+import java.util.List;
+
+public class Converter implements FxConversionService {
+
+    private ExternalQuotesService externalQuotes;
+
+    public Converter(ExternalQuotesService externalQuotesService){
+        externalQuotes = externalQuotesService;
+    }
+
+    @Override
+    public BigDecimal convert(ClientOperation operation, Symbol symbol, BigDecimal amount) {
+        List<Quote> quotes = externalQuotes.getQuotes(symbol);
+        Volume volume = Volume.from(amount);
+
+        if(volume.isInfinity()) throw new FxConversionException("Сумма меньше или равна 0");
```

Comment:
&#39;if&#39; construct must use &#39;{}&#39;s.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-50945078>*